hack/ci/lib/lib.sh

```diff
@@ -36,9 +36,18 @@ DOCKER_REPO="quay.io/jetstack"
 # other purpose
 DOCKER_TAG="build"
 
-function kubeVersion() {
-    echo $(docker run \
-        --entrypoint="cat" \
-        "${KIND_IMAGE}" \
-        /kind/version)
-}
+if [ ! "${CM_DEPS_LOADED:-}" = "1" ]; then
+    # Build all e2e test dependencies
+    bazel build \
+        //hack/bin:helm \
+        //test/e2e:e2e.test \
+        //test/e2e:images \
+        "${KIND_TARGET}"
```

Comment:
The 'KIND_TARGET' here is what dictates which 'kind_cluster' target to build/run.